nautobot/core/testing/views.py

```diff
@@ -255,9 +255,16 @@ class GetObjectChangelogViewTestCase(ModelViewTestCase):
 
         @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
         def test_get_object_changelog(self):
-            url = self._get_url("changelog", self._get_queryset().first())
+            obj = self._get_queryset().first()
+            url = self._get_url("changelog", obj)
             response = self.client.get(url)
             self.assertHttpStatus(response, 200)
+            response_data = response.content.decode(response.charset)
+            if type(obj) not in [extras_models.Contact, extras_models.Team]:
+                self.assertIn(
```

Comment:
Probably better to use `assertInHTML` here and below (https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.SimpleTestCase.assertInHTML)